docs/.vuepress/config.js

```diff
@@ -358,6 +358,14 @@ module.exports = {
           add('keywords', $site.themeConfig.keywords)
         }
       }
+    ],
+    [
+      'vuepress-plugin-canonical',
+      {
+        // add <link rel="canonical" header (https://tools.ietf.org/html/rfc6596)
+        // to deduplicate SEO across all copies loaded from various public gateways
+        baseURL: 'https://docs.ipfs.io'
```

Comment:
I assume this should reflect the primary site access point, such as `docs-beta.ipfs.io` then `docs.ipfs.io` once the original site is deprecated?

For the meantime I've added a rule to stop robots indexing the site while it's in beta, however we might decide to change that once we've rearranged a few sections.

Reply:
Yes, if we want to deploy this to `docs-beta.ipfs.io` first, then we should use it:

```suggestion
        baseURL: 'https://docs-beta.ipfs.io'
```